### DISCUSSION TITLE
Class 'org.oppia-android.app.profile.AddProfileActivityTest' not found in mudule 'oppia-android.app.unitTest'

### DISCUSSION THREAD

**Discussion by ShubhadeepKarmakar** [oppia/oppia-android#4830] [Q&A (Other)]
![oppia testing](https://user-images.githubusercontent.com/99060332/211977915-8d9235b2-c93e-4c5e-b95e-761ecbac774f.png)

**Comment by MohitGupta121** [comment]
@ShubhadeepKarmakar Which Android Studio version you are using?
Recommended version for oppia-android to run Tests is - **Android Studio Bumblebee (Patch 3)** as it's also works for me and also you refer to this https://github.com/oppia/oppia-android/discussions/4432.

If it fixed by this answer please mark this as answer or if you find any other way that work for you please share and mark that as answer. So that this helps others in community.

**Comment by Rd4dev** [comment]
Hi @ShubhadeepKarmakar, 

As @MohitGupta121 Mohit pointed out, I was able to successfully run the Robo-electric tests on the **Android Studio Bumblebee (Patch 3)** version by following [these instructions](https://github.com/oppia/oppia-android/wiki/Installing-Oppia-Android#configure-robolectric-tests) and using **Java 11**!

![Roboelectric tests working](https://github.com/oppia/oppia-android/assets/122200035/df87f50a-6ae8-4948-9a1b-62c0212f28f2)
